dashboard/app/models/foorm/library.rb

```diff
@@ -48,6 +48,9 @@ def self.setup
         library.published = published
         library.save! if library.changed?
 
+        # remove existing questions to catch any that were removed in the file
+        library.library_questions.destroy_all
+
         source_questions["pages"].map do |page|
           page["elements"].map do |element|
             question_name = element["name"]
```

Comment:
There's no finding or initializing here any more if we destroy all of them beforehand, yeah? Would be nice to update this to `new` along with this change to make that clear.

There was some discussion when we started doing this seeding/reseeding of Forms/etc. of trying to use `find_or_initialize_by` in order to have continuity in the database IDs that represented each Form (or Library Question). Trying to dig up why that was a preferred approach, and if there's any downside to just deleting them all...